api/types/build.go

```diff
@@ -191,6 +192,7 @@ func (b *Build) Environment(workspace, channel string) map[string]string {
 		envs["VELA_PULL_REQUEST"] = number
 		envs["VELA_PULL_REQUEST_SOURCE"] = b.GetHeadRef()
 		envs["VELA_PULL_REQUEST_TARGET"] = b.GetBaseRef()
+		envs["VELA_PULL_REQUEST_FORK"] = ToString(b.GetFork())
```

Comment:
is this info attached to pull requests only? or can we do `VELA_REPO_FORK` and report it on non-PR events too

Reply:
ohhh i understand what you mean, yeah i get it. sweet